server/src/test/java/com/paypal/selion/proxy/SeLionSauceProxyTest.java

```diff
@@ -0,0 +1,66 @@
+package com.paypal.selion.proxy;
+
+import com.paypal.selion.pojos.SeLionGridConstants;
+import org.apache.commons.io.FileUtils;
+import org.openqa.grid.common.RegistrationRequest;
+import org.openqa.grid.internal.RemoteProxy;
+import org.openqa.grid.internal.utils.GridHubConfiguration;
+import org.openqa.grid.selenium.proxy.DefaultRemoteProxy;
+import org.openqa.grid.web.Hub;
+import org.openqa.selenium.remote.internal.HttpClientFactory;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.InputStream;
+
+import static org.testng.Assert.assertTrue;
+
+public class SeLionSauceProxyTest {
+
+    private HttpClientFactory httpClientFactory;
+    private Hub hub;
+    private RegistrationRequest req;
+    private File tempFile;
+
+    @BeforeClass
+    public void setup() throws Exception {
+        httpClientFactory = new HttpClientFactory();
+        String[] args = new String[] {
+                "-role", "hub", "-type", "sauce",
+                "-host", "localhost",
+                "-servlets", "com.paypal.selion.grid.servlets.LoginServlet,com.paypal.selion.grid.servlets.SauceServlet",
+        };
+        GridHubConfiguration ghc = GridHubConfiguration.build(args);
+        hub = new Hub(ghc);
+        // use config files from resource location instead of default ~/.selionHome
+        System.setProperty("selionHome", (new File("src/main/resources")).getAbsolutePath());
```

Comment:
What about saving the original value and then restoring in afterClass? 
origSelionHome = System.setProperty("selionHome", (new File("src/main/resources")).getAbsolutePath());